### DISCUSSION TITLE
Cytoscape resource

### DISCUSSION THREAD

**Discussion by yogeshkodei** [cytoscape/cytoscape.js#2818] [General]
Greetings:
We are looking for a developer who can help improvise our cytoscape implementation. We have issues with some layouts and usage of the browser real estate. Can you please let me know where I could find one?

**Comment by Bokomoko** [comment]
I can help you

**Reply by yogeshkodei** [reply]
That’s great. Can you please tell me more about yourself? Can we talk sometime soon?

**Reply by Bokomoko** [reply]
I'm a computer professional with 30+ years of software development experience. You can view my profile on[ Linkedin ](https://www.linkedin.com/in/joaoeurico). Since last February I´ve been using Cytoscape.JS for developing some cool app´s for my current job. You can reach me by email (joao.eurico@live.com), skype (joao.eurico), whatsapp, telegram.

**Reply by yogeshkodei** [reply]
Great! What time would be convenient? Would like to talk to you tomorrow and give you a demo of our application and see how you can help. Would also like to see some of your work.

**Reply by Bokomoko** [reply]
I´ll be available from 10:30 AM until noon. My time zone is UTC -3. What´s yours?

**Reply by yogeshkodei** [reply]
I am in California, USA. Time now here is 2:22 m and in Brazil it is 6:22 pm

**Reply by yogeshkodei** [reply]
There is a 4 hour difference. But we could talk. Will send you a meeting invite

**Reply by yogeshkodei** [reply]
Your email bounced back.

**Reply by yogeshkodei** [reply]
Is there an alternative way of reaching you by email?

**Reply by Bokomoko** [reply]
try bokomoko@gmail.com

**Comment by yogeshkodei** [comment]
sent
 
--------- Original Message --------- Subject: Re: [cytoscape/cytoscape.js] Cytoscape resource (#2818)
From: "Bokomoko" ***@***.***>
Date: 8/23/21 6:10 pm
To: "cytoscape/cytoscape.js" ***@***.***>
Cc: "yogeshkodei" ***@***.***>, "Author" ***@***.***>

  
 try ***@***.***
 -
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
